app/src/main/java/com/yml/womensafety/navigationdrawer/home/HomePageActivity.kt

```diff
@@ -0,0 +1,109 @@
+package com.yml.womensafety.navigationdrawer.home
+
+import android.annotation.SuppressLint
+import android.content.Intent
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import androidx.fragment.app.FragmentTransaction
+import com.google.android.material.bottomnavigation.BottomNavigationView
+import com.google.android.material.snackbar.Snackbar
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.database.*
+import com.yml.womensafety.authentication.LoginActivity
+import com.yml.womensafety.R
+import com.yml.womensafety.navigationdrawer.TipsToEscapeFragment
+import com.yml.womensafety.navigationdrawer.contacts.ContactsFragment
+import com.yml.womensafety.navigationdrawer.youtube.SelfDefenseVideoFragment
+import kotlinx.android.synthetic.main.activity_home_page.*
+
+class HomePageActivity : AppCompatActivity(){
+    private lateinit var auth: FirebaseAuth
+    private var databaseReference: DatabaseReference? = null
+    private var firebaseDatabase: FirebaseDatabase? = null
+    private lateinit var homePageFragment: HomePageFragment
+    private lateinit var contactsFragment: ContactsFragment
+    private lateinit var tipsToEscapeFragment: TipsToEscapeFragment
+    private lateinit var selfDefenseVideoFragment: SelfDefenseVideoFragment
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_home_page)
+        val bottomNavigationView = findViewById<BottomNavigationView>(R.id.bottomNavBar)
+        bottomNavigationView.background = null
```

Comment:
Use apply scope function and set the values to BottomNavigationView

Reply:
done